cassandra-mesos-scheduler/pom.xml

```diff
@@ -80,6 +80,11 @@
             <groupId>com.fasterxml.jackson.datatype</groupId>
             <artifactId>jackson-datatype-guava</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Jackson is already in the code base, please use that rather than adding another json library.